src/material/_index.scss

```diff
@@ -0,0 +1,88 @@
+// Theming APIs
+@forward './core/theming/theming' show define-light-theme, define-dark-theme,
```

Comment:
Technically we don't _need_ to list out all the APIs with `show`, but I liked being explicit about what we expose. I'm open towards changing it since this could result in us forgetting to re-export a new API.

Reply:
+1, I like this being explicit as well; from here it's easy to see our entire Sass API surface